INN/umbrella-sfpublicpress
# Make secondary homepage featured post flex so image is centered

## Changes

This pull request makes the following changes:

- Makes the secondary featured post on the homepage `flex` so that the image is centered with the text.

Before:
![Screen Shot 2020-07-06 at 10 56 08 AM](https://user-images.githubusercontent.com/18353636/86607560-68e3e280-bf77-11ea-87e3-6d7f12e8e7cb.png)

After:
![Screen Shot 2020-07-06 at 10 55 58 AM](https://user-images.githubusercontent.com/18353636/86607572-6b463c80-bf77-11ea-98f6-d341569c2250.png)

## Why



For #152 

## Testing/Questions

Features that this PR affects:

- Homepage secondary featured post


Questions that need to be answered before merging:


Steps to test this PR:

1. View the secondary featured post area on the homepage and make sure it looks ok
